src/Tribe/Tickets_Handler.php

```diff
@@ -93,9 +98,10 @@ public function __construct() {
 		foreach ( $main->post_types() as $post_type ) {
 			add_action( 'save_post_' . $post_type, array( $this, 'save_image_header' ) );
 			add_action( 'save_post_' . $post_type, array( $this, 'save_global_stock' ) );
-			add_action( 'save_post_' . $post_type, array( $this, 'save_tickets_order' ) );
 		}
 
+		add_action( 'save_post_' . Tribe__Events__Main::POSTTYPE, array( $this, 'save_tickets_order' ) );
```

Comment:
That was not the intention, was done in response to some looping issues. Corrected.